templates/includes/sidebar.html

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+{#  <p>Navigation:</p>#}
+  <ul class="list-group">
+    <li class="list-group-item">
+      <a href="{% url 'taxi:index' %}">Home page</a>
+    </li>
+    <li class="list-group-item">
+      <a href="#">Manufacturers</a>
+    </li>
+    <li class="list-group-item">
+      <a href="#">Cars</a>
+    </li>
+    <li class="list-group-item">
+      <a href="#">Drivers</a>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are placeholders. Consider updating these with actual URLs or removing them if they are not needed.